book/src/csi-windows.md

```diff
@@ -0,0 +1,99 @@
+
+Please note that CSI controller level operations/sidecars are not supported on Windows.
+
+## How to use the CSI Proxy for Windows?
+See how to install CSI Proxy in [csi-proxy.md#Deployment]
+
+For CSI driver authors, import CSI proxy client under github.com/kubernetes-csi/csi-proxy/client. There are six client API groups including disk, filesystem, iscsi, smb, system, volume. See [link](https://github.com/kubernetes-csi/csi-proxy/tree/master/client/groups) for details.
+As an example, please check how GCE PD Driver import disk, volume and filesystem client API groups [here](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/release-1.2/pkg/mount-manager/safe-mounter_windows.go#L28)
+
+The Daemonset specification of a CSI node plugin for Windows can mount the desired named pipes from CSI Proxy based on the version of the API groups that the node-plugin needs to execute.
+
+
+The following Daemonset YAML shows how to mount various API groups from CSI Proxy into a CSI Node plugin:
+
+```
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-storage-node-win
+spec:
+  selector:
+    matchLabels:
+      app: csi-driver-win
+  template:
+    metadata:
+      labels:
+        app: csi-driver-win
+    spec:
+      serviceAccountName: csi-node-sa
+      nodeSelector:
+        kubernetes.io/os: windows
+      containers:
+        - name: csi-driver-registrar
+          image: k8s.gcr.io/sig-storage/csi-node-driver-registrar
+          args:
+            - "--v=5"
+            - "--csi-address=unix://C:\\csi\\csi.sock"
+            - "--kubelet-registration-path=C:\\kubelet\\plugins\\plugin.csi\\csi.sock"
+          volumeMounts:
+            - name: plugin-dir
+              mountPath: C:\csi
+            - name: registration-dir
+              mountPath: C:\registration
+        - name: csi-driver
+          image: k8s.gcr.io/sig-storage/csi-driver:win-v1
+          args:
+            - "--v=5"
+            - "--endpoint=unix:/csi/csi.sock"
+          volumeMounts:
+            - name: kubelet-dir
+              mountPath: C:\var\lib\kubelet
+            - name: plugin-dir
+              mountPath: C:\csi
+            - name: csi-proxy-disk-pipe
+              mountPath: \\.\pipe\csi-proxy-disk-v1beta2
```

Comment:
Let's update the api versions in this example to the latest - v1

Reply:
done